libr/bin/format/mdmp/mdmp.c

```diff
@@ -521,7 +546,10 @@ static bool r_bin_mdmp_init_directory_entry(struct r_bin_mdmp_obj *obj, struct m
 		break;
 	case COMMENT_STREAM_W:
 		/* TODO: Not yet fully parsed or utilised */
-		obj->streams.comments_w = obj->b->buf + entry->location.rva;
+		obj->streams.comments_w = r_buf_get_at (obj->b, entry->location.rva, NULL);
```

Comment:
using r_buf_get_at () is a bad idea because this only works with some RBuffer backends. so, for example if you are trying to parse an MDMP from an IO RBuffer (which only uses a file descriptor to operate), , the r_buf_get_at will return NULL and fail. The reason to keep r_buf_get_at is to have a way to get a pointer inside the buffer when we know for sure that the backend supports it.

Reply:
What do you suggest as a general solution?

Reply:
wel i would merge this as is, because the solution is to have a wrapper api that holds temporal buffers or copy all the time